config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
I think that right now this is hitting semaphore, right? If so, a PR should be added on `tests` to install `q35` qemu instead of the currently `qemu-lite`.
Although I am not sure if we are providing it as a package in OBS.